tests/snapshots/tests.contrib.django.test_django_snapshots.test_appsec_enabled.json

```diff
@@ -19,12 +19,14 @@
       "django.response.class": "django.http.response.HttpResponse",
       "django.user.is_authenticated": "False",
       "django.view": "tests.contrib.django.views.index",
+      "http.client_ip": "",
```

Comment:
should this be an empty string?

should we not set it if we don't have a value?

Reply:
Maybe, but we're already returning the empty string for this tag in previous versions (probably went for empty because there are other empty strings in the jsons) so I don't want to change it now.